webchild_sampling.py

```diff
@@ -10,37 +10,41 @@
 from mowgli.lib.etl.pipeline_wrapper import PipelineWrapper
 from mowgli.lib.etl.web_child.web_child_pipeline import WebChildPipeline
 from mowgli.lib.etl.pipeline_storage import PipelineStorage
-
+from mowgli.lib.etl.web_child.web_child_extractor import WebChildExtractor
+from mowgli.paths import DATA_DIR, PROJECT_ROOT
+from os import listdir
 
 
 def loadcsv():
 
-    print("Downloading Zip")
-    url = urlopen('http://people.mpi-inf.mpg.de/~ntandon/resources/relations/partOf/webchild_partof.zip')
-    urlstream = BytesIO(url.read())
+    part_whole_zip_url = 'http://people.mpi-inf.mpg.de/~ntandon/resources/relations/partOf/webchild_partof.zip'
```

Comment:
The WebChildExtractor __init__ takes default parameters, you don't need to pass it any.